pom.xml

```
@@ -545,6 +620,13 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <configuration>
+                    <forkCount>0</forkCount>
```

```suggestion
                    <useModulePath>false</useModulePath>
```
This fixes the root cause